src/stactools/datacube/stac.py

```diff
@@ -76,6 +76,32 @@ def get_time_offset_and_step(unit: str) -> Tuple[datetime, timedelta]:
     raise ValueError(f"Failed to parse time unit from '{unit}'")
 
 
+def iso_duration(td: timedelta) -> str:
```

Comment:
Can you add a simple docstring describing the function?

Reply:
Done